src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -420,7 +438,45 @@ export class ServiceFormComponent extends CdForm implements OnInit {
         ]
       ],
       grafana_port: [null, [CdValidators.number(false)]],
-      grafana_admin_password: [null]
+      grafana_admin_password: [null],
+      // oauth2-proxy
+      https_address: [null, [
+        CdValidators.requiredIf({
+          service_type: 'oauth2-proxy'
+        }),
+        CdValidators.custom('httpsAddressPattern', (value: string) => {
+          if (_.isEmpty(value)) {
+            return false;
+          }
+          if (!value.includes(':')) {
+            return true;
+          }
+          const [address, port] = value.split(':');
+          const addressTest = this.OAUTH2_HTTPS_ADDRES_PATTERN.test(address);
+          const portTest = this.OAUTH2_HTTPS_PORT_PATTERN.test(port);
+          return !(addressTest && portTest);
+        })
+      ]],
+      provider_display_name: [null, [
+        CdValidators.requiredIf({
+          service_type: 'oauth2-proxy'
+        })
```

Comment:
we can have default value here "My OIDC provider"

Reply:
Not a fan of arbitrary default values. 
But I can include it of course

Reply:
Added it